src/cpp/server/server_context.cc

```diff
@@ -146,4 +148,8 @@ bool ServerContext::IsCancelled() {
   return completion_op_ && completion_op_->CheckCancelled(cq_);
 }
 
+std::unique_ptr<const AuthContext> ServerContext::auth_context() const {
+  return CreateAuthContext(call_);
```

Comment:
just:
`return auth_context_;`
where `auth_context_` is a `shared_ptr<const AuthContext>`.